docs/use-the-network/coverage-mapping/coverage-mapping.mdx

```diff
@@ -5,5 +5,18 @@ sidebar_label: Coverage Mapping
 slug: /use-the-network/coverage-mapping
 ---
 
-## Coverage Mapping
+# Coverage Mapping
 
+Helium recently began an effort to collect device data for the purpose of
+coverage mapping from voluntary community contributors. Mapping data is viewable
+at [mappers.helium.com](https://mappers.helium.com/). Anyone can contribute to
+this project by sending device uplinks with the required geographical data to
+the [Mappers API](/docs/use-the-network/coverage-mapping/mappers-api). For a step by step guide on setting an
+integration in Console see the
+[Mappers Quickstart](/docs/use-the-network/coverage-mapping/mappers-quickstart).
+
+[Mappers API](/docs/use-the-network/coverage-mapping/mappers-api).
+
+[Mappers Data](/docs/use-the-network/coverage-mapping/mappers-data).
+
+[Adeunis Mapper Instructions](/docs/use-the-network/coverage-mapping/adeunis-mapper).
```

Comment:
I would break this whole section out into a list after `...data to
the [Mappers API]...` Maybe something like:

```suggestion

## Go Deeper with Coverage Mapping

* Read the step-by-step [Mappers Quickstart](/docs/use-the-network/coverage-mapping/mappers-quickstart)
* Learn more about the [Mappers API](/docs/use-the-network/coverage-mapping/mappers-api)
* Get some [Mappers Data](/docs/use-the-network/coverage-mapping/mappers-data)
* Deploy the [Adeunis Mapper](/docs/use-the-network/coverage-mapping/adeunis-mapper)

```